### PR TITLE
🎨 MT-1147 legislation box style adjustments

### PR DESCRIPTION
**Pull request by itemodahl**
- prevent icons being cut off

MT-1147

### CODE REVIEW

File: src/scss/components/_disclosure.scss [hunk 1]
```diff
@@ -114,11 +115,11 @@
         flex: none;
         align-self: center;
         width: $menu-icon-width;
-        height: $menu-icon-height-mobile;
+        height: 100%;
       }
       @media only screen and (min-width: $breakpoint-large) {
```
Comment (lha047) [inline src/scss/components/_disclosure.scss:120]: Dont need the media query if it doesnt change the value

**Review by lha047 — CHANGES_REQUESTED** [review]
Left some suggestions :)

File: src/scss/components/_disclosure.scss [hunk 2]
```diff
@@ -141,6 +142,8 @@
       transform-origin: center center;
     }
     .disclosure-panel {
+      @include children-spaced(var(--spacer-xx-small));
```
[the remark this answers is not in the document]
Comment (itemodahl) [reply src/scss/components/_disclosure.scss:145]: The spacing is wanted on all content in disclosures. Ill ask Malin for clarification.

[the remark this answers is not in the document]
Comment (itemodahl) [reply src/scss/components/_disclosure.scss]: After a brief talk with Lisa, we concluded that this css rule will only apply for lovdata content. It is moved to the mattilsynet-web project.

File: src/scss/components/_disclosure.scss (hunk 1 above)
Comment (itemodahl) [reply src/scss/components/_disclosure.scss]: removed

**Review by lha047 — APPROVED** [review]
LGTM :D

**Pull request merged**